qualang_tools/voltage_gates/README.md

```diff
@@ -0,0 +1,202 @@
+# VoltageGateSequence
+
+The `VoltageGateSequence` class facilitates the creation and management of complex pulse sequences for quantum operations, allowing for dynamic voltage control, ramping, and compensation across multiple gate elements.
+
+> **Note**: Make sure to initialize the class before opening the `qm`!
+
+> **Note**: Make sure the check the timing of the pulses in simulation or use `strict_timing()` to avoid real-time gaps!
+
+## Features
+
+- **Dynamic Voltage Control**: Define voltage levels (points) in gate-space and durations for multiple elements.
+- **Step and Ramp Support**: Add steps and ramps to the pulse sequence.
+- **Compensation Pulse**: Keeps track of average voltage levels for compensation pulses.
+- **Configuration Management**: Updates the provided configuration with necessary operations and waveforms for the sequence.
```

Comment:
I am not sure, but maybe write Keep and Update to be compatible with the infinitive of the bullet points above.

Reply:
Done